stable/aws-load-balancer-controller/templates/deployment.yaml

```diff
@@ -90,6 +90,12 @@ spec:
         {{- if .Values.watchNamespace }}
         - --watch-namespace={{ .Values.watchNamespace }}
         {{- end }}
+        {{- if index .Values "election-id" }}
+          - --election-id={{ .Values index "election-id" }}
```

Comment:
You will need to update the resource name in the rbac as well here [1] 

[1] https://github.com/aws/eks-charts/blob/master/stable/aws-load-balancer-controller/templates/rbac.yaml#L14

Reply:
I have added the condition for using this value in RBAC rules if that value has been set